### ISSUE TITLE
Add a proper dark mode

### ISSUE THREAD

**Issue by SaturnDreemurr32X** [sirjuddington/SLADE#1739]
### Editor

Resource editor

### Description of your feature request

I would like SLADE to ... again whats in the subject, AN ACTUAL DARK MODE BECAUSE ITS 12 AM AND I AM GETTING FLASHED BY THE LIGHT THERE WASNT A PROPER DARK MODE AND MY EYES KEEP HURTING
![Screenshot 2024-10-09 004327](https://github.com/user-attachments/assets/df0d1a83-d2e9-4eee-9ad8-856b06c0e648)
PLEASE add a proper dark mode because my eyes hurt every single day i open this and im not joking

### Mock-up

Just replacing the light to the dark

**Comment by SaturnDreemurr32X** [comment]
Apologies with me complaining about that its just really difficult using this at day and night it just really hurts my eyes.

**Comment by sirjuddington** [comment]
This is also a duplicate of #1188. The status is still the same, waiting on wx 3.3 to release which supposedly will have dark mode support
